mod/quiz/classes/question/bank/custom_view.php

```diff
@@ -198,68 +207,4 @@ protected function create_new_question_form($category, $canadd): void {
     protected function display_question_bank_header(): void {
     }
 
-    protected function build_query(): void {
```

Comment:
why is this removed? this will introduce regression where quiz should only use ready questions not draft or hidden

Reply:
I will review and add it back